readthedocs/rtd_tests/tests/test_backend.py

```diff
@@ -57,6 +57,19 @@ def test_git_checkout(self):
         repo.checkout()
         self.assertTrue(exists(repo.working_dir))
 
+    def test_git_tags(self):
+        repo_path = self.project.repo
+        create_tag(repo_path, 'v01')
+        create_tag(repo_path, 'v02', annotated=True)
+        create_tag(repo_path, 'release-ünîø∂é')
+        repo = self.project.vcs_repo()
+        # Hack the repo path
+        repo.working_dir = repo_path
```

Comment:
Why do we need to do this here?

Reply:
Because we don't make the `clone` part, so the vcs object will try to search the repo in `/project/checkouts/...`. Sorry for the late reply, I was focused in fixin the unicode problem :grin: